releng/lib/check_artifacts.py

```diff
@@ -164,9 +164,13 @@ def do_check_docker(checker: Checker, name: str) -> None:
                     else:
                         assert_eq(subcheck.result, ga_ver)
 
-    with checker.check(name='Git tags') as check:
-        check.result = 'TODO'
-        raise NotImplementedError()
+    # This is redundant since we now look at the tag for which the GitHub release was
+    # created -- we're trusting GitHub not to allow a release that points to a tag that
+    # doesn't exist.
+    # with checker.check(name='Git tags') as check:
+    #     check.result = 'TODO'
+    #     raise NotImplementedError()
+        
```

Comment:
Can we just delete this and add a comment on the GH release check saying that "the existence of a release implies the existence of the tag (though you can actually then delete the tag without deleting the release)"?

Reply:
Done.